app/components/containerConsole/containerConsole.html

```diff
@@ -33,8 +33,13 @@
                 </select>
               </div>
             </div>
+            <div class="col-sm-3">
+              <div class="input-group">
+                <input class="form-control" type="text" name="user" ng-model="state.user" placeholder="User">
+              </div>
+            </div>
             <!-- !command-list -->
-            <div class="col-sm-8">
+            <div class="col-sm-5">
```

Comment:
Ok, I thought about more "central" layout in this case, please have a look and let me know what do you think:
<img width="1027" alt="screen shot 2017-07-04 at 16 11 03" src="https://user-images.githubusercontent.com/6943514/27831443-6c87bb2e-60d3-11e7-80c1-f8dc30880c81.png">